src/main/java/no/unit/nva/doi/transformer/model/crossrefmodel/CrossrefApiResponse.java

```diff
@@ -0,0 +1,48 @@
+package no.unit.nva.doi.transformer.model.crossrefmodel;
+
```

Comment:
Nva-doi-lambda and nva-doi-fetch, they just add a location header and forward the response from Crossref as it is. So, we need to parse the original response from Crossref, which contains some other additional info